luarules/gadgets/scav_spawner_defense.lua

```diff
@@ -185,19 +185,11 @@ if gadgetHandler:IsSyncedCode() then
 	-- Teams
 	--------------------------------------------------------------------------------
 
-	local teams = GetTeamList()
-	for _, teamID in ipairs(teams) do
-		local teamLuaAI = GetTeamLuaAI(teamID)
-		if (teamLuaAI and string.find(teamLuaAI, "ScavengersAI")) then
-			scavTeamID = teamID
-			scavAllyTeamID = select(6, Spring.GetTeamInfo(scavTeamID))
-			--computerTeams[teamID] = true
-		else
-			humanTeams[teamID] = true
-		end
-	end
+	local scavTeamID = Spring.Utilities.GetScavTeamID()
+
+	local scavAllyTeamID = Spring.Utilities.GetScavAllyTeamID() 
 
-	local gaiaTeamID = GetGaiaTeamID()
+	local gaiaTeamID = GetGaiaTeamID() --??
```

Comment:
```suggestion
	local gaiaTeamID = Spring.GetGaiaTeamID()
```
And remove the localization on line 39. This function is called once on loading the gadget file, the localization is completely pointless.